includes/Core/Authentication/Google_Proxy.php

```diff
@@ -95,6 +95,72 @@ public function get_site_fields() {
 		);
 	}
 
+	/**
+	 * Fetch site fields
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Credentials $credentials Credentials instance.
+	 * @return array|WP_Error The response as an associative array or WP_Error on failure.
+	 */
+	public function fetch_site_fields( Credentials $credentials ) {
+		if ( ! $credentials->has() ) {
+			return new WP_Error( 'oauth_credentials_not_exist' );
```

Comment:
@benbowler, please, add `use WP_Error;` to the top of the class to make it work here.